src/components/messageThread.css

```diff
@@ -0,0 +1,24 @@
+.sent {
+  position: relative;
+  background: #74c2ff;
+  border-radius: 15px;
+}
+.sentContainer {
+}
+.messageContent {
+  position: relative;
+  width: 755px;
+  font-family: Arial;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 150%;
```

Comment:
why is your line height 150% here? You shouldn't even have to mess with line heights.